spec/features/realtime_data/outgoing_resgistrations/index_outgoing_registrations_spec.rb

```diff
@@ -17,4 +19,18 @@
     expect(page).to_not have_css('flash-warning')
     expect(page).to_not have_css('flash-error')
   end
+
+  context 'with additional fields' do
+    before do
+      stub_jrpc_request('show.registrations', node.rpc_endpoint).and_return([record_filled.stringify_keys])
+      visit outgoing_registrations_path(q: { node_id_eq: node.id })
+    end
+
+    it 'has record' do
+      expect(page).to have_css('.col-id', text: record_filled[:id])
+      expect(page).to have_css('.col-last_reply_contacts')
+      expect(page).to have_css('.col-last_request_contact')
+      expect(page).to have_css('.col-pop', text: pop.name)
+    end
+  end
```

Comment:
you don't need to add `before` in this context.
`before` from parent `context/describe` will be executed in nested context

in this context you need to override `record_attributes` and check that additional fields are shown correctly in table

like this
```ruby
  context 'with additional fields' do
    let(:record_attributes) do
      attributes_for(:outgoing_registration, :filled, pop: pop, last_reply_contacts: 'foo', last_request_contact: 'bar')
    end

    it 'has record' do
      expect(page).to have_css('.col-id', text: record_attributes[:id])
      expect(page).to have_css('.col-last_reply_contacts', text: record_attributes[:last_reply_contacts])
      expect(page).to have_css('.col-last_request_contact', text: record_attributes[:last_request_contact])
      expect(page).to have_css('.col-pop', text: pop.name)
    end
  end
```

Reply:
Should I check the value of the added fields?

Reply:
yes

Reply:
Can't fount  pop in table and in record_attributes Hash missing also ()
part from output error rspec:
`but there were no matches. Also found "pop", ""`
byebug:
record_attributes.key?(:pop_id) #=> false

Reply:
```
def initialize(nodes, pops, params = {})
  @params = clean_search_params(params).with_indifferent_access
  @errors = []
  set_nodes(nodes)
  set_pops(pops)
end
  
```

Reply:
Maybe I should add parameter for initialize OutgoingRegistrations